core/templates/pages/collection-player-page/collection-player-page-root.component.ts

```diff
@@ -32,4 +32,9 @@ export class CollectionPlayerPageRootComponent extends BaseRootComponent {
 
   meta: MetaTagData[] = AppConstants.PAGES_REGISTERED_WITH_FRONTEND
     .COLLECTION_PLAYER.META as unknown as Readonly<MetaTagData>[];
+  ngOnInit(): void {
```

Comment:
Seems reasonable, but can you verify if the title is still 'Loading...' while it hasn't loaded yet? Thanks!